lib/email_downcaser.rb

```diff
@@ -0,0 +1,8 @@
+class EmailDowncaser
+  def self.dump(value)
+    value.downcase
+  end
+  def self.load(value)
+    value
+  end
+end
```

Comment:
Not sure why we're down casing email addresses - technically the part before the @ can be case sensitive.